.gitpod/run-phpstorm.sh

```diff
@@ -3,4 +3,4 @@
 if [ ! -x ~/.projector/configs/PhpStorm/run.sh ]; then
   echo "PhpStorm runner not found" && exit 1
 fi
-~/.projector/configs/PhpStorm/run.sh $GITPOD_REPO_ROOT
+~/.projector/configs/PhpStorm/run.sh "$GITPOD_REPO_ROOT"
```

Comment:
Hadn't seen this before, didn't know you could do this.  Now we just need a way to specify the license key so you don't have to do that all the time.